### ISSUE TITLE
Repositories with multiple RBFs

### ISSUE THREAD

**Issue by sorgelig** [MiSTer-devel/Updater_script_MiSTer#37]
There is already one repository (Atari 800) with multiple RBFs. And i'm not excluding that other similar repos may appear later.
Instead of processing it specifically i propose a generic automated way. The RBF names equal to .qsf names in the root. So, basically you scan the root for qsf files and you already have possible RBFs in release folder.
Atari800.qsf -> Atari800_YYYYMMDD.rbf
Atari5200.qsf -> Atari5200_YYYYMMDD.rbf

**Comment by Locutus73** [comment]
Hi this could be an idea to explore, but I see some problems here to think about:

1. The only core with two distinct RBF files in the releases folder is the Atari 800 one which would still need custom code because Atari800 rbf must go into _Computer dir, while Atari5200 rbf must go into _Console. Parsing qsf files would't help with this, it would be useful for cores with multiple rbf in the same category.
2. Some cores don't have the qsf file in the root of the repo, but in a different dir, i.e. PDP1 has the qsf in /src, Apple-I under /boards/MiSTer.
3. Sometimes qsf and rbf case don't match, i.e. sharpmz vs SharpMZ, but this isn't a real problem, I would search ignoring case.
4. Sometimes there are more than one qsf, but only one rbf, i.e. Minimig.qsf and Minimig_Q13.qsf vs Minimig rbf; this isn't a real problem too.

1. and 2. are the biggest problems I see in this approach, but maybe we can find a solution.

**Comment by sorgelig** [comment]
2. I will look into those repositories.
3. rbf name always matches the qsf name. The difference in cases because it's how qsf file is listed inside qpf. Since windows doesn't care of file cases, it may mix it.
4. this is not a problem. If script can't find a matching rbf, then simply ignore it.

**Comment by Locutus73** [comment]
Yeah, 3 and 4 aren't real problems, 1 and 2 are more interesting to study.
For 2 I gave a pair of repos as example, I'm not sure if there are (or aren't) others with strange directory structure.
